### PR TITLE
Use sanitized-users view

### PR DESCRIPTION
**Pull request by Jss7268**
This view was added for simplicity, so let's use it :)

### CODE REVIEW

**Comment by Jss7268** [comment]
@iamnukee this fixes the devices_experiments table too

File: models/user.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ module.exports = (db, Validator, UserAccess) => {
           else if (data.email) {
             findOneByEmail(data.email)
               .then((result) => {
-                delete result.password;
+                delete result.hashed_password;
```
Comment (Jss7268) [inline models/user.js]: this was a bug

Comment (iamnukee) [reply models/user.js]: Ahh, I see 👍

**Comment by iamnukee** [comment]
^^ need to resolve conflicts though

**Pull request merged**